Makefile

```diff
@@ -33,7 +33,7 @@ clean:
 	docker run --rm -ti -e JEKYLL_UID=`id -u` -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll clean
 
 htmlproofer: clean _site
-	docker run -ti -e JEKYLL_UID=`id -u` --rm -v $$PWD/_site:/_site/ quay.io/calico/htmlproofer /_site --file-ignore ${HP_IGNORE_LOCAL_DIRS} --assume-extension --check-html --empty-alt-ignore --url-ignore "/docs.openshift.org/,#,/github.com\/projectcalico\/calico\/releases\/download/"
+	docker run -ti -e JEKYLL_UID=`id -u` --rm -v $$PWD/_site:/_site/ quay.io/calico/htmlproofer /_site --file-ignore ${HP_IGNORE_LOCAL_DIRS} --assume-extension --check-html --empty-alt-ignore --url-ignore "/docs.openshift.org/,#,/github.com\/projectcalico\/calico\/releases\/download/,https://github.com/projectcalico/calico/tree/master/v3.0/getting-started/openshift/installation.md"
```

Comment:
This needs to be backed out before merging, right?

Reply:
Once this is resolved, LGTM

Reply:
YEs, I have to make new PR that removes the ignore, post merge

Reply:
Because I do not have the power to override this check. Unless you want to do it.

Reply:
If lets take this out and then I can override the merge. We expect it to pass once in master, right?

Reply:
All right, I've pushed up this change. Yes, it is a chicken egg problem with the proofer. Sounds good 👍